flink-runtime-web/web-dashboard/src/app/pages/job/exceptions/job-exceptions.component.less

```diff
@@ -35,8 +40,8 @@
   }
 
   nz-code-editor {
-    position: relative;
-    height: calc(~"100vh - 310px");
+    position: absolute;
```

Comment:
Have you tried this in relevant browsers? I would have expected this to need something like

```
top: 0;
left: 0;
bottom: 0;
right: 0;
```

Or does nz-code-editor detect and handle this?